### PR TITLE
Install installer #8934 ke

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/8934

### CODE REVIEW

File: modules/installation/pages/general-requirements.adoc [hunk 1]
```diff
@@ -42,15 +42,20 @@ Depending on your organization's setup, you might also need to activate your sub
 
 
 [[install.media]]
-== Obtaining Installation Media
+== Obtaining {slsa} Unified Installer
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]: is the {slsa} really necessary here? Also, it should probably be "Obtain"

File: modules/installation/pages/general-requirements.adoc [hunk 2]
```diff
@@ -42,15 +42,20 @@ Depending on your organization's setup, you might also need to activate your sub
...
-From {slsa}{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy is available as a base product, and can be installed with the {slsa} Unified Installer.
+From {slsa}{nbsp}15{nbsp}SP1, {susemgr} {productnumber} Server and Proxy is available as a base product, and can be installed with the {slsa} Unified Installer.
 
 ifeval::[{suma-content} == true]
-Therefore you do not require a separate registration code for {slsa} {sles-version} {sp-version}, only for Server or Proxy
+Therefore you do not require a separate registration code for {slsa}, only for {susemgr} Server or Proxy.
 endif::[]
 
-Download {slsa}{nbsp}15{nbsp}SP1 or higher from https://www.suse.com/products/server/download/[SUSE Linux Enterprise Server - Media Download]
+If not already done, download the {slsa} Unified Installer
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion
Download the {slsa} Unified Installer
```

File: modules/installation/pages/general-requirements.adoc [hunk 3]
```diff
@@ -42,15 +42,20 @@ Depending on your organization's setup, you might also need to activate your sub
...
+(version {nbsp}15{nbsp}SP1 or higher) from
+https://www.suse.com/products/server/download/[SUSE Linux Enterprise
+Server - Media Download].  With the Unified Installer you can installf
+many SLE-based base products such as {slsa}, {sled}, {slsa} for SAP
+Applications, or {susemgr}.
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]: Line breaks here are weird.

File: modules/installation/pages/general-requirements.adoc (hunk 1 above)
Comment (keichwa) [reply modules/installation/pages/general-requirements.adoc]: Maybe {sle}, maybe not at all. Yes.

File: modules/installation/pages/general-requirements.adoc (hunk 3 above)
Comment (keichwa) [reply modules/installation/pages/general-requirements.adoc]: I probably pressed Alt-q (M-q) ;-(

**Comment by keichwa** [comment]
all proposals are in.

**Pull request merged**